src/sentry/api/event_search.py

```diff
@@ -1546,10 +1546,14 @@ def has_default(self, params):
         ],
         "result_type": "integer",
     },
-    "divide": {
-        "name": "divide",
+    "percentage": {
+        "name": "percentage",
         "args": [FunctionArg("numerator"), FunctionArg("denominator")],
-        "aggregate": [u"divide", [ArgValue("numerator"), ArgValue("denominator")], None],
+        "aggregate": [
+            u"if(greater({denominator},0),divide({numerator},{denominator}),null)",
```

Comment:
This is an edge case, and can be done later. If the duration changes from 0ms to 100s between the two time periods, this will be reported as `NULL` and will not be surfaced. This is more problematic with `user_misery` where 0 miserable users in a time period can be common.